cmd/profile/delete.go

```diff
@@ -33,6 +43,13 @@ func deleteProfileCommand() *cobra.Command {
 }
 
 func deleteProfile(config *restclient.Config, ns string, profileName string) error {
+	ctrl, err := controller.NewClient(config)
+	errors.CheckError(err)
```

Comment:
It doesn't make sense to fatally fail the error, if the function is designed to return an error code. So if NewClient() fails, just return something like `fmt.Errorf("failed to get client: %s", err)`

Reply:
`errors.CheckError` does that. From the docs:
> CheckError logs a fatal message and exits with ErrorGeneric if err is not nil

Reply:
done